wordpress-mobile/WordPress-Android
# Issue/13268 introduce date range picker screen in Activity Log

Partially fixes #13268 

Merge instructions: 
1. Review this PR
2. Make sure #13270 is merged
3. Remove "Not ready for merge" label
4. Merge this PR

This PR introduces Material Fullscreen Date Range picker component in Activity Log. It overrides some styles of the material component to facilitate future styling. The design of the date range picker hasn't been finalised yet so ignore the overall styling for now.

Unit tests will be added in a separate PR.

To test:
1. My Site -> Avatar icon in the toolbar -> App Settings -> Test feature configuration
2. Enable ActivityLogFiltersFeatureConfig and click on "Restart app"
3. Open Activity Log
4. Click on the "Date range" chip
5. Notice the fullscreen date range picker is shown
6. Select a date range
7. Click on "Cancel"
8. Click on the "Date range" chip and notice no date range is pre-selected
9. Select a date range
10. Click on "Save"
11. Click on the "Date range" chip and notice the date range from step 9 is pre-selected 

PR submission checklist:
